ansible-profile/callback_plugins/collect_info.py

```diff
@@ -0,0 +1,193 @@
+# The MIT License (MIT)
```

Comment:
Probably should keep using Apache2 software license?

Reply:
Fixed